HIP/hip-820.md

```diff
@@ -71,38 +71,34 @@ When a dApp requires only the signature from the controller (wallet), it can use
+### hedera_signAndExecuteTransaction
+When a dApp needs the services of the controller to sign a transaction message and then submit it to the network, it can use the `hedera_signTransactionAndSend` method.  This method accepts a base64 encoded protobuf representation of the Hedera API [`TransactionList`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/sdk/transaction_list.proto) message as input.  The controller decodes the message, interprets the contents, and if valid and approved, appends it’s signature to the [`SignatureMap`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/basic_types.proto#L780) structure identified as the [sigMap](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L40C13-L40C13) property of the [`SignedTransaction`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L31) repackages the message as appropriate for transmission to the Hedera network and sends it to the network.  The controller shall wait for the initial response from the Hedera node and return the resulting Node Precheck Code generated by the submission.
 
 The controller is allowed to retry submitting the signed transaction to the Hedera Network for certain types of error conditions; such as gRPC connection droppage or Hedera `BUSY` signals.  This specification does not obligate the controller to retry, but recommends the practice to help improve perceived user experience for consumer applications.
 
 #### Parameters
-`signedTransaction` – a base64 encoding of the protobuf endoded Hedera API [`SignedTransaction`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L31) message.  The message must include the [bodyBytes](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L35C11-L35C11) field representing the transaction, but inclusion of a [sigMap](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/services/transaction_contents.proto#L40C13-L40C13) is optional.  If included, and if it is pre-populated with signatures, the pre-existing signatures must also be included in the message sent to the Hedera Node.
+`transaction` – a base64 encoding of the protobuf endoded Hedera API [`TransactionList`](https://github.com/hashgraph/hedera-protobufs/blob/f36e05bd6bf3f572707ca9bb338f5ad6421a4241/sdk/transaction_list.proto) message.
```

Comment:
`TransactionList` is not a HAPI construct, it is an artifact of the Hedera JS SDK.  This object is not utilized by all of the SDKs.  One of the goals for this protocol is to be SDK agnostic.  This works against that goal.  Furthermore, this is a vestiture of pre-open access thinking.  We do not want to ask a wallet to sign multiple duplicate transactions and pay the network fees to send those transactions.  The `TransactionList` and `Transaction` protobuf messages are not necessary as the `SignedTransaction` contains just enough information to complete the task and nothing extra.

Reply:
@bugbytesinc the intention is to be able to utilize the `fromBytes` methods in the SDKs to easily reconstruct a transaction. This should be transferable between sdks.

a list is utilized in go, java, and javascript in one way or another. If there's a better description of the payload that can also utilize these methods, please let me know, thanks!

`go` - https://github.com/hashgraph/hedera-sdk-go/blob/d3bf16d478e3a1226d2078ebdc6a7a0d5232c5e1/transaction.go#L95

`java` - https://github.com/hashgraph/hedera-sdk-java/blob/7ffc43474cd1c8532cd3d0d7a4ef740a618bd697/sdk/src/main/java/com/hedera/hashgraph/sdk/Transaction.java#L239

`javascript` - https://github.com/hashgraph/hedera-sdk-js/blob/fbcca47f9653bc0f53edb56b116300f48082bde1/src/transaction/Transaction.js#L197

The intention here is for ease of onboarding, vs. the smallest possible payload

Let's have a straw poll here as well to get other's feedback. Thumbs up on the preferred way of doing this